src/main/java/com/exactpro/epfast/template/xml/TemplateXml.java

```diff
@@ -0,0 +1,81 @@
+package com.exactpro.epfast.template.xml;
+
+import com.exactpro.epfast.template.Dictionary;
+import com.exactpro.epfast.template.Identity;
+import com.exactpro.epfast.template.Template;
+import com.exactpro.epfast.template.xml.helper.AfterUnmarshal;
+import com.exactpro.epfast.template.xml.helper.Namespace;
+import com.exactpro.epfast.template.xml.helper.NsXmlParent;
+import com.exactpro.epfast.template.xml.helper.TemplateNsXmlParent;
+
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "template", namespace = Namespace.XML_NAMESPACE)
+public class TemplateXml extends InstructionsXml implements Template, NsXmlParent {
+
+    private AfterUnmarshal.TemplateIdentity templateId = new AfterUnmarshal.TemplateIdentity();
+
+    private String ns;
+
+    private Dictionary dictionary;
+
+    private ReferenceImpl typeRef;
+
+    @Override
+    public Identity getTemplateId() {
+        return templateId;
+    }
+
+    @XmlAttribute(name = "name")
+    public void setName(String name) {
+        this.templateId.setName(name);
+    }
+
+    @XmlAttribute(name = "templateNs")
+    public void setTemplateNs(String templateNs) {
+        this.templateId.setNamespace(templateNs);
+    }
+
+    @XmlAttribute(name = "id")
+    public void setId(String id) {
+        this.templateId.setAuxiliaryId(id);
+    }
+
+    @Override
+    public String getNs() {
+        return ns;
+    }
+
+    @XmlAttribute(name = "ns")
+    public void setNs(String ns) {
```

Comment:
Names for xml parsing should keep naming closer to xml names. So this name is OK.